From e2355df47f7679e3a7ecc5e1200b58d024fbb2b1 Mon Sep 17 00:00:00 2001
From: Jingtao Chen <jingtaoc@codeaurora.org>
Date: Wed, 17 Feb 2016 17:04:44 +0800
Subject: [PATCH 55/67] APBU-11490 bugfix

For some divx asf files, there is no framerate in the caps after demux.
Remove the framerate in gst-omx conf then it can connect omxmpeg4videodec in the pipeline.

Change-Id: If284cf4cf89e7e418669bd1ad4355e05bd9d787d
Signed-off-by: Jingtao Chen <jingtaoc@codeaurora.org>
---
 parsed_data_gstomx.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/parsed_data_gstomx.conf b/parsed_data_gstomx.conf
index a9264d0..1e40f5f 100755
--- a/parsed_data_gstomx.conf
+++ b/parsed_data_gstomx.conf
@@ -14,7 +14,7 @@ component-name=OMX.IMG.MSVDX.Decoder.MPEG4
 rank=260
 in-port-index=0
 out-port-index=1
-sink-template-caps=video/mpeg, mpegversion = (int) 4, parsed = (boolean) false, profile = (string) simple, systemstream = (boolean) false, level = (string) {0, 1, 2, 3}, width = (int) [16, 4096], height = (int) [16, 4096], framerate = (fraction) [0/1, 2147483647/1]; video/mpeg, mpegversion = (int) 4, parsed = (boolean) false, profile = (string) advanced, systemstream = (boolean) false, level = (string) {0, 1, 2, 3, 4, 5}, width = (int)[16, 4096], height = (int) [16, 4096], framerate = (fraction) [0/1, 2147483647/1]; video/mpeg, mpegversion = (int) 4, parsed = (boolean) true, systemstream = (boolean) false; video/x-divx, divxversion = (int)[ 3, 5 ], width = (int) [16, 4096], height = (int) [16, 4096], framerate = (fraction) [0/1, 2147483647/1]; video/x-xvid, width = (int) [16, 4096], height = (int) [16, 4096], framerate = (fraction) [0/1, 2147483647/1]; video/x-3ivx, width = (int) [16, 4096], height = (int) [16, 4096], framerate = (fraction) [0/1, 2147483647/1]; video/x-msmpeg, msmpegversion = (int) 4, width = (int) [16, 4096], height = (int) [16, 4096], framerate = (fraction) [0/1, 2147483647/1]; video/x-msmpeg, msmpegversion = (int)[ 41, 43 ], width = (int) [16, 4096], height = (int) [16, 4096], framerate = (fraction) [0/1, 2147483647/1]
+sink-template-caps=video/mpeg, mpegversion = (int) 4, parsed = (boolean) false, profile = (string) simple, systemstream = (boolean) false, level = (string) {0, 1, 2, 3}, width = (int) [16, 4096], height = (int) [16, 4096], framerate = (fraction) [0/1, 2147483647/1]; video/mpeg, mpegversion = (int) 4, parsed = (boolean) false, profile = (string) advanced, systemstream = (boolean) false, level = (string) {0, 1, 2, 3, 4, 5}, width = (int)[16, 4096], height = (int) [16, 4096], framerate = (fraction) [0/1, 2147483647/1]; video/mpeg, mpegversion = (int) 4, parsed = (boolean) true, systemstream = (boolean) false; video/x-divx, divxversion = (int)[ 3, 5 ], width = (int) [16, 4096], height = (int) [16, 4096]; video/x-xvid, width = (int) [16, 4096], height = (int) [16, 4096], framerate = (fraction) [0/1, 2147483647/1]; video/x-3ivx, width = (int) [16, 4096], height = (int) [16, 4096], framerate = (fraction) [0/1, 2147483647/1]; video/x-msmpeg, msmpegversion = (int) 4, width = (int) [16, 4096], height = (int) [16, 4096], framerate = (fraction) [0/1, 2147483647/1]; video/x-msmpeg, msmpegversion = (int)[ 41, 43 ], width = (int) [16, 4096], height = (int) [16, 4096], framerate = (fraction) [0/1, 2147483647/1]
 
 [omxmpeg2videodec]
 type-name=GstOMXMPEG2VideoDec
-- 
1.9.1

